uniffi_bindgen/src/interface/object.rs

```diff
@@ -232,6 +232,7 @@ impl Constructor {
         self.ffi_func.name.push_str("_");
         self.ffi_func.name.push_str(&self.name);
         self.ffi_func.arguments = self.arguments.iter().map(Into::into).collect();
+        // XXX - we need this to be some kind of pointer abstraction - `usize`?
```

Comment:
If we're going to work with raw pointers, I think the right thing to do here is add an `FFIType::Pointer` variant and then map it explicitly to the corresponding pointer abstractions in each foreign language. (The existing `FFITypeRustCString` variant actually already is *sort* of this, but specifically for a pointer-to-char).